recipes-extended/jailhouse/jailhouse_0.10.bb

```diff
@@ -0,0 +1,11 @@
+require jailhouse.inc
+
+SRCBRANCH = "imx_4.19.35_1.1.0"
+IMX_JAILHOUSE_SRC ?= "git://source.codeaurora.org/external/imx/imx-jailhouse.git;protocol=ssh"
+
+SRC_URI = "${IMX_JAILHOUSE_SRC};branch=${SRCBRANCH}"
+SRCREV = "e83ab629ccc6f44e4caa05bc51c66d2f86546ee6" 
+
+CELLS = ""
+
+COMPATIBLE_MACHINE = "${@bb.utils.contains('MACHINE_FEATURES', 'jailhouse', '${MACHINE}', '(^$)', d)}"
```

Comment:
No reason for a .inc and a .bb for such small difference. Please squash the .inc and .bb so it is easier to follow and maintain.

Reply:
Done